src/viam/sdk/components/movement_sensor/client.cpp

```diff
@@ -26,90 +27,71 @@ MovementSensorClient::MovementSensorClient(std::string name, std::shared_ptr<grp
 using namespace viam::component::movementsensor::v1;
 
 Vector3 MovementSensorClient::get_linear_velocity(const AttributeMap& extra) {
-    GetLinearVelocityResponse response;
-    stub_wrapper<GetLinearVelocityRequest>(
```

Comment:
One of the big advantages of `ClientHelper` over `stub_wrapper` is that you only need to know the function on the stub, and you don't need to name the request and response types explicitly (though you can, if you want, on the lambda).